pkgs/development/python-modules/pki-core/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, buildPythonPackage, sphinx, rpmextract, requests,
+  six }:
+
+buildPythonPackage rec {
+  name = "pki-core-${version}";
```

Comment:
```suggestion
  pname = "pki-core";
```

Reply:
Removed pki-core entirely, as it is not in use.